src/main.rs

```diff
@@ -99,6 +102,8 @@ enum Commands {
         properties: Vec<String>,
         #[clap(short, long)]
         messages: Option<u64>,
+        #[clap(long, default_value_t = 33)]
+        client_spawn_timeout_ms: u64,
```

Comment:
So for 10,000 clients we'd take 5.5mins to start all of them with the 33ms default? Is there a reason for the high-ish default value? 

Nit: Also timeout seems a bit of a misnomer. There isn't exactly a process we're giving up on after the timeout. It's more of a wait.

Reply:
> So for 10,000 clients we'd take 5.5mins to start all of them with the 33ms default? Is there a reason for the high-ish default value?

Yes, 5 minutes doesn't seem too long given the test would wait ~30 minutes anyway. It being more conservative is purely to make it more stable.

With this config I was able to run the GCP t9 tests on a lot less hardware than what duck.py would chose automatically. (n2d-standard-32 vs n2-highmem-80).

I will change the name to wait.

Reply:
Lets see.

> A fix could be fencing all the clients after they complete their first operation. Then releasing the fence once all clients are created.

I mean this is already broken in most of the tests anyway so we will need this later in any case indpendent of this change.